app/code/Magento/Store/etc/config.xml

```diff
@@ -22,6 +22,7 @@
                 <minify_files>0</minify_files>
                 <minify_exclude>
                     <tiny_mce>/tiny_mce/</tiny_mce>
```

Comment:
shouldn't we remove the old "/tiny_mce/"?

Reply:
Not sure, it appears we still have files that match that pattern (no idea if they actually still being used, didn't spend time researching that yet):
```
$ find . -type f -path '*/tiny_mce/*'
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/tinymceAdapter.js
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/plugins/magentovariable/editor_plugin.js
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/plugins/magentowidget/editor_plugin.js
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/setup.js
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/html5-schema.js
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/themes/advanced/skins/default/content.css
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/themes/advanced/skins/default/dialog.css
./lib/web/mage/adminhtml/wysiwyg/tiny_mce/themes/ui.css
```